po/common/ro.po

```diff
@@ -4858,7 +4858,7 @@ msgstr "Detalii despre calculul Scorului Eco"
 # do not translate Eco-score
 msgctxt "ecoscore_information"
 msgid "Information about the Eco-score"
-msgstr "Informații despre Eco-Score"
+msgstr "Informații despre Scorul Eco"
```

Comment:
@stephanegigandet no, I can spot mistakes like this one that need to be corrected in Crowdin